src/main/java/hobbylist/ui/MainWindow.java

```diff
@@ -146,8 +153,52 @@ public void handleHelp() {
             helpWindow.focus();
         }
     }
+    /**
+     * Change primaryStage to star theme.
+     */
+    @FXML
+    public void handleStar() {
```

Comment:
would repeated calls to this method cause multiple duplicate stylesheets to be added?

Reply:
Thank you I have tried to fix